README.md

```diff
@@ -85,30 +85,40 @@ Now in the server repository, push your server application to Cloud Foundry
+Once the deployment is succeeds, your backend server should be running on the cloud and able to classify the different kinds of waste once the model finishes training. Please take a note with your server application's endpoint as you will need it in the next step. Now let's go ahead and create our mobile app to use this classifier.
 
-## 3. Create the mobile application and connect it with the server
+## 3. Create the mobile application and connect it to the server
 
-## Backend API usage
+In order to test the full features for this application, you need to have [Xcode 8.0 and above](https://developer.apple.com/xcode/) installed and an IOS device to deploy the application.
 
-Do a POST request at `https://watson-waste-sorter.mybluemix.net/api/sort` with the image as the parameter. 
-Return value should be in JSON.
+Now Open your Xcode and select `Open another project...`, then select the `mobile-app/WatsonWasteSorter.xcworkspace` file and click `Open`.
 
-Example in Bash:
+Next, you need to modify the `WatsonWasteSorter/Info.plist` with the endpoint of the API server you just deployed. Replace the `SERVER_API_ENDPOINT`'s value section
+with your server endpoint with extension `/api/sort`.
 
-Input: png/jpg/jpeg file
-```
-curl -X POST -F "images_file=@server/plastic_fork.jpg" "https://watson-waste-sorter.mybluemix.net/api/sort"
-```
+![plist](docs/plist.png)
 
-Output: 
-```
-{"confident score": 0.547405, "status code": 200, "result": "Recycle"}
-```
+Next, you will need to sign your application with your Apple account. Go to the mobile app's `General` section, under `Signing`'s Team select your team or add an account. Now your mobile app is signed and you are ready to deploy your Waste Sorter app.
+
+> Note: If you have trouble to Sign your Mobile app, please refer to https://help.apple.com/xcode/mac/current/#/dev60b6fbbc7
 
+Now, Connect your IOS device to your machine and select your device in Xcode. Click the run icon and your mobile app should be installed on your device.
+
+## 4. Using the Waste Sorter mobile application
+
+Congratulation, at this point you should have a mobile app that can classify wastes using your camera. Now you can just simply point your camera to any waste
+and click the camera icon to take a picture. Then the application should tell you where the waste should go like this.
+
+![screenshot](docs/screenshot.jpg)
+
+Now you should have a better idea on how to sort your trash. Note that if you have a result that said `unclassified`, it means your image is too blurry or the
+waste is too far. In that case just simply point your camera closer and retake a new picture.
+> If you want to classify another waste, simply click the center of the screen.
 
 # Troubleshooting
 
+- To clean up, simply simple delete your mobile app and run `cf delete watson-waste-sorter` to delete your server.
```

Comment:
is that hyphen a bullet? oh my.  Steve really prefers * for bullets.  :)

We normally would not use the cf command here.  Our typical practice is more IBM Cloud focused.  First you want an app w/ a timestamped name so that people don't get name collisions.  Next we would probably just tell them to use the IBM Cloud UI to delete the app (but honestly we tend to not document clean up at all).